app/reducers/devices.js

```diff
@@ -6,6 +6,7 @@ const devices = (state = {}, action) => {
     case 'DEVICE_ADDED':
     case 'DEVICE_REMOVED':
     case 'DEVICE_UPDATED':
+    case 'DEVICE_DISCOVERED':
```

Comment:
is it really necessary to have a seperate action for `DEVICE_DISCOVERED`, why cant you use `DEVICE_ADDED`?